### PR TITLE
R-igraphdata: various improvements

### PR DESCRIPTION
**Pull request by szhorvat**
- set R.author so it's GitHub-compatible
 - disable testing, since there are no tests
 - correct license

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by szhorvat** [comment]
CC @barracuda156

**Comment by barracuda156** [comment]
I am not sure re tests, since we *can* run check on it, regardless of whether a port author bothered to write tests so far.

**Comment by barracuda156** [comment]
@szhorvat By the way, why don’t you add yourself as a co-maintainer?

File: R/R-igraphdata/Portfile [hunk 1]
```diff
@@ -15,5 +15,3 @@ checksums           rmd160  01b953f881370ce443d0bd5b3e832238403e7532 \
 supported_archs     noarch
 
 depends_lib-append  port:R-igraph
-
-test.run            yes
```
Comment (barracuda156) [inline R/R-igraphdata/Portfile:19]: I would rather keep it. We would want to switch to `test.run yes` by default, eventually (well, IMO). If anything, setting `test.run yes` signals that all dependencies are in, and no additional work is needed. Also `R` allows to run checks for any package, whether special tests are included or not. Why would we prohibit that?

Comment (szhorvat) [reply R/R-igraphdata/Portfile]: Re-added. Let's merge.

Comment (barracuda156) [reply R/R-igraphdata/Portfile]: LGTM

**Pull request merged**